apps/frontend/src/libs/enums/success-message.enum.ts

```diff
@@ -0,0 +1,5 @@
+const SuccessMessage = {
```

Comment:
Let's rename it to NotificationMessage. We will use it for toast notifications, whereas ExceptionMessage is for Error class notifications only
```suggestion
const NotificationMessage = {
```